internal/consensus/reactor.go

```diff
@@ -794,10 +794,10 @@ func (r *Reactor) gossipVotesRoutine(ctx context.Context, ps *PeerState, voteCh
 		// catchup logic -- if peer is lagging by more than 1, send Commit
 		blockStoreBase := r.state.blockStore.Base()
 		if blockStoreBase > 0 && prs.Height != 0 && rs.Height >= prs.Height+2 && prs.Height >= blockStoreBase {
-			// Load the block commit for prs.Height, which contains precommit
+			// Load the block's extended commit for prs.Height, which contains precommit
 			// signatures for prs.Height.
-			if commit := r.state.blockStore.LoadBlockCommit(prs.Height); commit != nil {
-				if ok, err := r.pickSendVote(ctx, ps, commit, voteCh); err != nil {
+			if extCommit := r.state.blockStore.LoadBlockExtCommit(prs.Height); extCommit != nil {
+				if ok, err := r.pickSendVote(ctx, ps, extCommit, voteCh); err != nil {
```

Comment:
For a short-lived local, a shorter name would be fine:

```suggestion
			if ec := r.state.blockStore.LoadBlockExtCommit(prs.Height); ec != nil {
				if ok, err := r.pickSendVote(ctx, ps, ec, voteCh); err != nil {
```

YMMV